team-agreements/minimalism-rules.md

```diff
@@ -1,6 +1,6 @@
-# Follow Minimalism Rules 
+# Follow Minimalism Rules
 
-## (http://minifesto.org/)
+## (<http://minifesto.org/)>
```

Comment:
While we’re fixing things, can we remove the ## here - this doesn’t make sense as a title, we can just leave it as a link

Reply:
I'm only running autofixing in this PR.